README.md

```diff
@@ -109,6 +108,4 @@ Videos and Presentations
 - [JavaOne 2017: Eclipse OpenJ9 Under the hood of the next open source JVM](https://www.slideshare.net/DanHeidinga/javaone-2017-eclipse-openj9-under-the-hood-of-the-jvm)
 - [JavaOne 2017: Ask the OpenJ9 Architects](https://www.youtube.com/watch?v=qb5ennM_pgc)
 
-Also check out the [Resources](https://www.eclipse.org/openj9/oj9_resources.html) page on our website.
-
 Copyright (c) 2017, 2018 IBM Corp. and others
```

Comment:
Pls update the copyright end year here, and at the top of the page.

Reply:
Good spot. Fixed.